.gitlab-ci.yml

```diff
@@ -143,3 +143,20 @@ publish_public_latest:
     IMG_SOURCES: $BUILD_DOCKER_REGISTRY/$PROJECTNAME:$CI_COMMIT_TAG-amd64,$BUILD_DOCKER_REGISTRY/$PROJECTNAME:$CI_COMMIT_TAG-arm64
     IMG_DESTINATIONS: operator:latest
     IMG_SIGNING: "false"
+
+trigger_internal_image:
+  stage: release
+  rules:
```

Comment:
I think you should have

```
  rules:
    - if: $CI_COMMIT_TAG
    - when: never
 ```

And `needs` on `build_image_arm64` and `build_image_amd64` otherwise it won't trigger automatically on tag

Reply:
thanks